packages/block-editor/src/components/inserter/library.js

```diff
@@ -24,6 +24,7 @@ function InserterLibrary(
 		__experimentalOnPatternCategorySelection,
 		onSelect = noop,
 		shouldFocusBlock = false,
+		onClose,
```

Comment:
It's not great to pass the props here, but I can't find a way to access this state without passing it down.

Reply:
This will get undone when we refactor the List View and Inserter to share the same sidebar, so I think it's OK.